bootstrap/kubeadm/types/utils.go

```diff
@@ -51,18 +51,11 @@ var (
 	}
 )
 
-func KubeVersionToKubeadmAPIGroupVersion(version string) (schema.GroupVersion, error) {
-	if version == "" {
-		return schema.GroupVersion{}, errors.New("version cannot be empty")
-	}
-	semVersion, err := versionutil.ParseSemantic(version)
-	if err != nil {
-		return schema.GroupVersion{}, errors.Wrap(err, "error parsing the Kubernetes version")
-	}
+func KubeVersionToKubeadmAPIGroupVersion(version semver.Version) (schema.GroupVersion, error) {
 	switch {
-	case semVersion.LessThan(versionutil.MustParseSemantic("v1.13.0")):
+	case version.LT(semver.MustParse("1.13.0")):
```

Comment:
Should we make these package variables so we only use MustParse once during the package initialization?